### PR TITLE
Making descText and confirmText not required for all confirm modals

### LINKED ISSUE #79: Fix confirm modal PropTypes
`ConfirmDeleteQueueModal` and friends currently use the same propTypes as `ConfirmModal`, but they should only use a subset since `descText` and `confirmText` are never provided as props to those components.

### PR DESCRIPTION
**Pull request by josh-byster**
Fixed alongside @PradyumnaShome and @jrogge.

Fixes #79 and issue in #121.

### CODE REVIEW

**Comment by nwalters512** [comment]
Thanks, I'll review this later!

FYI, you don't have to keep making new PRs, you can push to the same branch to automatically update the existing one :)

**Comment by josh-byster** [comment]
Thanks Nathan! Quick question regarding that: since I wanted to fork Prad's changes, would I still have been able to do this and reopen with the pull request from my original fork at #115? Would I fetch the latest master and then rebase Prad's changes over that? Although then the issue is Prad's branch and mine diverged at some point. I'm pretty comfortable with Git but I wasn't sure how this would work.

**Comment by nwalters512** [comment]
Ah, didn't realize you were working separately. That does make it a bit tricky - I assumed you were working from the same fork. In general, when collaborating on a changeset with someone, you want to be working on the same repo.

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ with the current date and the next changes should go under a **[Next]** header.
 * Reorganize directory structure. ([@nwalters512](https://github.com/nwalters512) in [#108](https://github.com/illinois/queue/pull/108))
 * Remove location from notifications if the queue is a fixed-location queue. ([@redsn0w422](https://github.com/redsn0w422) in [#123](https://github.com/illinois/queue/pull/123))
 * Add `npm run fix-lint-js` to fix linter errors that can be fixed automatically. ([@redsn0w422](https://github.com/redsn0w422) in [#127](https://github.com/illinois/queue/pull/127))
+* Fixed confirm modal PropTypes to make descText and confirmText not required. ([@josh-byster](https://github.com/josh-byster), [@PradyumnaShome](https://github.com/PradyumnaShome), and [@jrogge](https://github.com/jrogge) in [#130](https://github.com/illinois/queue/pull/130))
```
Comment (nwalters512) [inline CHANGELOG.md]: Nitpicks: present tense verbs and backticks on `descText` and `confirmText` to mark them as code

**Comment by nwalters512** [comment]
I have GitHub set up to automatically squash when merging, so no need!

I like to keep the changelog ordered chronologically by merge time, so if you could move you changelog to the bottom of the list, we'll be good to go!

**Pull request merged**